meta-lmp-base/recipes-bsp/u-boot/u-boot-fio_2022.04.bb

```diff
@@ -1,5 +1,6 @@
 require u-boot-fio-common.inc
 
-SRCREV = "d724ad15e2155b7373ea8b710ed85a7f141c2cb1"
+UBOOT_REPO = "git://github.com/ricardosalveti/u-boot.git"
```

Comment:
I think it's better to merge u-boot changes to the foundries repo and then bump u-boot from that repo.

Reply:
Yes, it's temporary just so we can test the entire set of changes needed.

Reply:
Updated.